src/components/PersonalPage/DevCard.tsx

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import {Card, Image, ListGroup} from 'react-bootstrap';
+import "./DevCard.css"
+const swarmLogo = require('../../images/swarmLogoIcon.png');
+
+/*
+    User will be the prop for this component, passed in from the personal page component.
+    This will provide the user info, and provide necessary info the query the info the dev page. ie username/id
+*/
+
+
+class DevCard extends React.Component<any, any>{
+    //Declaring props for the component
+    profileName: string;
+    gradYear: string;
+    degree: string;
+    teamRole: string;
+    bio: string;
+
+    //Setting the props
+    constructor(props: any){
```

Comment:
Props should not have any associated with it. I would recommend looking into user defined interfaces to specify what exactly the props will be receiving